From 08962a56f69963e01892d98ca5b75de8354bd3f5 Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Wed, 2 Nov 2011 16:54:57 +0100
Subject: [PATCH] Fix phar packaging

Inherited from OE-Classic, with some additions to fix host paths leaking
into the target package.

Upstream-Status: Inappropriate [config]

---
 ext/phar/Makefile.frag | 17 +++--------------
 1 file changed, 3 insertions(+), 14 deletions(-)

diff --git a/ext/phar/Makefile.frag b/ext/phar/Makefile.frag
index 0e3713f..22f7898 100644
--- a/ext/phar/Makefile.frag
+++ b/ext/phar/Makefile.frag
@@ -10,20 +10,9 @@ pharcmd: $(builddir)/phar.php $(builddir)/phar.phar
 
 PHP_PHARCMD_SETTINGS = -n -d 'open_basedir=' -d 'output_buffering=0' -d 'memory_limit=-1' -d phar.readonly=0 -d 'safe_mode=0'
 PHP_PHARCMD_EXECUTABLE = ` \
-	if test -x "$(top_builddir)/$(SAPI_CLI_PATH)"; then \
-		$(top_srcdir)/build/shtool echo -n -- "$(top_builddir)/$(SAPI_CLI_PATH) -n"; \
-		if test "x$(PHP_MODULES)" != "x"; then \
-		$(top_srcdir)/build/shtool echo -n -- " -d extension_dir=$(top_builddir)/modules"; \
-		for i in bz2 zlib phar; do \
-			if test -f "$(top_builddir)/modules/$$i.la"; then \
-				. $(top_builddir)/modules/$$i.la; $(top_srcdir)/build/shtool echo -n -- " -d extension=$$dlname"; \
-			fi; \
-		done; \
-		fi; \
-	else \
-		$(top_srcdir)/build/shtool echo -n -- "$(PHP_EXECUTABLE)"; \
-	fi;`
-PHP_PHARCMD_BANG = `$(top_srcdir)/build/shtool echo -n -- "$(INSTALL_ROOT)$(bindir)/$(program_prefix)php$(program_suffix)$(EXEEXT)";`
+		$(top_srcdir)/build/shtool echo -n -- "$(PHP_EXECUTABLE)"; `
+
+PHP_PHARCMD_BANG = `$(top_srcdir)/build/shtool echo -n -- "$(bindir)/$(program_prefix)php$(program_suffix)$(EXEEXT)";`
 
 $(builddir)/phar/phar.inc: $(srcdir)/phar/phar.inc
 	-@test -d $(builddir)/phar || mkdir $(builddir)/phar
